src/java/org/apache/cassandra/index/Index.java

```diff
@@ -751,6 +752,18 @@ interface Searcher
          * @return partitions from the base table matching the criteria of the search.
          */
         UnfilteredPartitionIterator search(ReadExecutionController executionController);
+
+        /**
+         * Returns the {@link Monitorable.ExecutionInfo} for this query, to be called by
+         * {@link ReadCommand#executionInfo()} at the end of the query to collect details about the query execution in
```

Comment:
same here, I would say let's call out in the docs what `details` mean for clarity.